### PR TITLE
EDU-998: update CLI docs

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?
This PR adds updates to the CLI documentation from the latest release.

### CODE REVIEW

File: docs-src/cli/cmd-options/metrics-port.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ tags:
     - command-line-interface-cli
     - logging and metrics
     - metrics
-
 ---
 
-Port for /metrics (default: 0).
+Port for `/metrics`.
+Disabled by default.
```
Comment (taonic) [inline docs-src/cli/cmd-options/metrics-port.md:16]: After some test I've verified the metrics endpoint is enabled by default but with a random port that's never advertised. I've created a issue here for the cli repo to track.

https://github.com/temporalio/cli/issues/309

Comment (DJSanti) [reply docs-src/cli/cmd-options/metrics-port.md:16]: Okay, I'll change this then.

**Review by feedmeapples — APPROVED** [review]

File: docs-src/cli/task-queue/get-build-id-reachability.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+tags:
+  - cli reference
+  - temporal cli
+  - task queue
+  - command-line-interface-cli
+---
+
+Determines if Build IDs may be used for for new, existing, or closed Workflows.
+Both the `--build-id` and `--task-queue` flags may be specified multiple times.
+Reachability for the provided Build IDs will be checked against all task queues if one isn't provided.
```
Comment (djmagee) [inline docs-src/cli/task-queue/get-build-id-reachability.md:15]:
```suggestion
Determines whether Build IDs can be used for for new, existing, or closed Workflows.
Both the `--build-id` and `--task-queue` options can be specified multiple times.
If a Task Queue isn't provided, reachability for the provided Build IDs is checked against all Task Queues.
```

File: docs-src/cli/task-queue/get-build-id-reachability.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+id: get-build-id-reachability
+title: temporal task-queue get-build-id-reachability
+sidebar_label: get-build-id-reachability
+description: Retrieves information about the reachability of Build IDs on one or more Task Queues
```
Comment (djmagee) [inline docs-src/cli/task-queue/get-build-id-reachability.md:5]:
```suggestion
description: Retrieves information about the reachability of Build IDs on one or more Task Queues.
```

File: docs-src/cli/task-queue/get-build-id-reachability.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (djmagee) [inline docs-src/cli/task-queue/get-build-id-reachability.md:1]: Add the new `temporal task-queue` subcommands (`get-build-id-reachability`, `get-build-ids`, and `update-build-ids`) to the `task-queue.json` config file.

File: docs-src/cli/task-queue/list-partition.md [hunk 5]
```diff
@@ -15,16 +15,34 @@ The `temporal task-queue list-partition` command displays the partitions of a [T
 
 Use the options listed below to change the command's behavior.
```
Comment (djmagee) [inline docs-src/cli/task-queue/list-partition.md:16]:
```suggestion
Use the following options to change the command's behavior.
```

File: docs-src/cli/workflow/update.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
+---
```
Comment (djmagee) [inline docs-src/cli/workflow/update.md:1]: Need to add this file to the `workflow.json` config file.

File: docs-src/cli/workflow/update.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+id: update
+title: temporal workflow update
+sidebar_label: update
+description: Updates a running workflow synchronously.
```
Comment (djmagee) [inline docs-src/cli/workflow/update.md:5]:
```suggestion
description: Updates a running Workflow Execution synchronously.
```

File: docs-src/cli/workflow/update.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+  - cli reference
+  - temporal cli
+  - workflow
+  - command-line-interface-cli
+  - workflow execution
+---
+
+The `temporal workflow update` command synchronously updates a running [Workflow Execution](/concepts/what-is-a-workflow-execution).
+
+Use the options listed below to change the command's behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/update.md:16]:
```suggestion
Use the following options to change the command's behavior.
```

File: docs-src/cli/workflow/count.md [hunk 9]
```diff
@@ -16,16 +16,32 @@ This command requires Elasticsearch to be enabled.
 
 Use the options listed below to change the command's behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/count.md:17]:
```suggestion
Use the following options to change the command's behavior.
```

File: docs-src/cli/workflow/delete.md [hunk 10]
```diff
@@ -15,18 +15,40 @@ The `temporal workflow delete` command deletes the specified [Workflow Execution
 
 Use the options listed below to change the command's behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/delete.md:16]:
```suggestion
Use the following options to change the command's behavior.
```

File: docs-src/cli/workflow/describe.md [hunk 11]
```diff
@@ -22,22 +22,38 @@ The output of this command can be changed to show as printed ('raw') or to only
 
 Use the command options listed below to change the information returned by this command.
```
Comment (djmagee) [inline docs-src/cli/workflow/describe.md:23]:
```suggestion
Use the following command options to change the information returned by this command.
```

File: docs-src/cli/workflow/index.md [hunk 12]
```diff
@@ -11,3 +11,6 @@ tags:
 ---
 
 [Workflow](/concepts/what-is-a-workflow) commands allow operations to be performed on [Workflow Executions](/concepts/what-is-a-workflow-execution).
+
+Workflow commands use this syntax:
+`temporal workflow COMMAND [ARGS]`
```
Comment (djmagee) [inline docs-src/cli/workflow/index.md:16]:
```suggestion
`temporal workflow COMMAND [ARGS]`.
```

File: docs-src/cli/workflow/list.md [hunk 13]
```diff
@@ -22,12 +22,28 @@ The command can also return a list of archived Workflow Executions.
 
 Use the command options listed below to change the information returned by this command.
```
Comment (djmagee) [inline docs-src/cli/workflow/list.md:23]:
```suggestion
Use thefollowing command options to change the information returned by this command.
```

File: docs-src/cli/workflow/query.md [hunk 14]
```diff
@@ -21,25 +21,41 @@ Queries can also be used on completed [Workflows](/concepts/what-is-a-workflow-e
 
 Use the command options listed below to change the information returned by this command.
```
Comment (djmagee) [inline docs-src/cli/workflow/query.md:22]:
```suggestion
Use the following command options to change the information returned by this command.
```

File: docs-src/cli/workflow/reset-batch.md [hunk 15]
```diff
@@ -25,28 +25,34 @@ Workflow Executions can also be found by [Query](/concepts/what-is-a-query).
 
 Use the options listed below to change reset behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/reset-batch.md:26]:
```suggestion
Use the following options to change reset behavior.
```

File: docs-src/cli/workflow/reset.md [hunk 16]
```diff
@@ -13,35 +13,51 @@ tags:
 ---
 
 The `temporal workflow reset` command resets a [Workflow Execution](/concepts/what-is-a-workflow-execution).
-A reset allows the Workflow to be resumed from a certain point without losing your parameters or [Event History](/concepts/what-is-an-event-history).
+A reset resumes the Workflow from a certain point without losing your parameters or [Event History](/concepts/what-is-an-event-history).
 
 The Workflow Execution can be set to a given [Event Type](/concepts/what-is-an-event).
 `temporal workflow reset --workflow-id=meaningful-business-id --type=LastContinuedAsNew`
```
Comment (djmagee) [inline docs-src/cli/workflow/reset.md:19]:
```suggestion
For example, `temporal workflow reset --workflow-id=meaningful-business-id --type=LastContinuedAsNew`.
```

File: docs-src/cli/workflow/reset.md [hunk 17]
```diff
@@ -13,35 +13,51 @@ tags:
...
 
-Alternatively, the Workflow Execution can be reset to any Event after WorkflowTaskStarted.
+The Workflow Execution can also be reset to any Event after WorkflowTaskStarted.
 `temporal workflow reset --workflow-id=meaningful-business-id --event-id=MyLastEvent`
```
Comment (djmagee) [inline docs-src/cli/workflow/reset.md:22]:
```suggestion
For example, `temporal workflow reset --workflow-id=meaningful-business-id --event-id=MyLastEvent`.
```

File: docs-src/cli/workflow/reset.md [hunk 18]
```diff
@@ -13,35 +13,51 @@ tags:
...
 
 Use the options listed below to change reset behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/reset.md:24]:
```suggestion
Use the following options to change reset behavior.
```

File: docs-src/cli/workflow/show.md [hunk 19]
```diff
@@ -12,24 +12,38 @@ tags:
   - event history
 ---
 
-The `temporal workflow show` command provides the [Event History](/concepts/what-is-an-event-history) for a specified [Workflow Execution](/concepts/what-is-a-workflow-execution).
+The `temporal workflow show` command provides the [Event History](/concepts/what-is-an-event-history) for a [Workflow Execution](/concepts/what-is-a-workflow-execution).
 
 Use the options listed below to change the command's behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/show.md:17]:
```suggestion
Use the following options to change the command's behavior.
```

File: docs-src/cli/workflow/signal.md [hunk 20]
```diff
@@ -13,33 +13,49 @@ tags:
   - signals
 ---
 
-The `temporal workflow signal` command is used to [Signal](/concepts/what-is-a-signal) a [Workflow Execution](/concepts/what-is-a-workflow-execution) by ID or [List Filter](/concepts/what-is-a-list-filter).
+The `temporal workflow signal` command is used to [Signal](/concepts/what-is-a-signal) a [Workflow Execution](/concepts/what-is-a-workflow-execution) by [ID](/concepts/what-is-a-workflow-id) or [List Filter](/concepts/what-is-a-list-filter).
```
Comment (djmagee) [inline docs-src/cli/workflow/signal.md:16]:
```suggestion
The `temporal workflow signal` command is used to send a [Signal](/concepts/what-is-a-signal) to a [Workflow Execution](/concepts/what-is-a-workflow-execution) by [Workflow Id](/concepts/what-is-a-workflow-id) or [List Filter](/concepts/what-is-a-list-filter).
```

File: docs-src/cli/workflow/signal.md [hunk 21]
```diff
@@ -13,33 +13,49 @@ tags:
...
 
 Use the options listed below to change the command's behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/signal.md:18]:
```suggestion
Use the following options to change the command's behavior.
```

File: docs-src/cli/workflow/stack.md [hunk 22]
```diff
@@ -18,24 +18,40 @@ Returning the stack trace of all the threads owned by a Workflow Execution can b
 
 Use the options listed below to change the command's behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/stack.md:19]:
```suggestion
Use the following options to change the command's behavior.
```

File: docs-src/cli/workflow/start.md [hunk 23]
```diff
@@ -19,12 +19,26 @@ When invoked successfully, the Workflow and Run ID are returned immediately afte
 
 Use the command options listed below to change how the Workflow Execution behaves upon starting.
```
Comment (djmagee) [inline docs-src/cli/workflow/start.md:20]:
```suggestion
Use the following command options to change how the Workflow Execution behaves upon starting.
```

File: docs-src/cli/workflow/terminate.md [hunk 24]
```diff
@@ -22,23 +22,39 @@ Workflow terminations require a valid [Workflow ID](/concepts/what-is-a-workflow
 
 Use the options listed below to change termination behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/terminate.md:23]:
```suggestion
Use the following options to change termination behavior.
```

File: docs-src/cli/workflow/trace.md [hunk 25]
```diff
@@ -17,26 +17,42 @@ The `temporal workflow trace` command tracks the progress of a [Workflow Executi
 
 Use the options listed below to change the command's behavior.
```
Comment (djmagee) [inline docs-src/cli/workflow/trace.md:18]:
```suggestion
Use the following options to change the command's behavior.
```

File: docs-src/cli/batch/index.md [hunk 26]
```diff
@@ -11,11 +11,8 @@ tags:
   - command-line-interface-cli
 ---
 
-Batch commands allow you to change multiple [Workflow Executions](/concepts/what-is-a-workflow-execution) in the background.
-In order to do this, you provide the command with a [List Filter](/concepts/what-is-a-list-filter) and the type of Batch job to execute.
-
-The List Filter identifies the Workflow Executions that will be affected by the Batch job.
-The Batch type determines the other parameters that need to be provided, along with what is being affected on the Workflow Executions.
+Batch commands change multiple [Workflow Executions](/concepts/what-is-a-workflow-execution) by providing a [List Filter](/concepts/what-is-visibility) and the type of Batch job to execute.
+The List Filter identifies the Workflow Executions in the Batch job; the Batch type determines what will happen to the Workflow Executions.
```
Comment (djmagee) [inline docs-src/cli/batch/index.md:15]:
```suggestion
Batch commands change multiple [Workflow Executions](/concepts/what-is-a-workflow-execution) by providing a [List Filter](/concepts/what-is-visibility) and the type of Batch Job to execute.
The List Filter identifies the Workflow Executions in the Batch Job; the Batch type determines what will happen to the Workflow Executions.
```

File: docs-src/cli/cmd-options/set-as-default.md [hunk 27]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (djmagee) [inline docs-src/cli/cmd-options/set-as-default.md:1]: Add to `cmd-options.md`. (Apologies for missing this one during my previous review.)

**Review by djmagee — COMMENTED** [review]
Just a couple of tweaks and one config change that I missed earlier.

**Review by djmagee — APPROVED** [review]

**Pull request merged**